### PR TITLE
Update web-nfc nfc_watch.https.html

### PR DESCRIPTION
**Pull request by xiuqijix**
part of #14389

@Honry, PTAL.

### CODE REVIEW

**Review by Honry — CHANGES_REQUESTED** [review]
You probably need to split this file into two, as triggering "reading" event requires user interaction with an NFC tag.

**Comment by kaixinjxq** [comment]
@Honry, many thanks for you review! I just submit a new commit to address your comments, PTAL.

File: web-nfc/NFCReader-manual.https.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Web NFC: NFCReader tests</title>
+<link rel="author" title="Intel" href="http://www.intel.com"/>
+<link rel="help" href="https://w3c.github.io/web-nfc/"/>
+<meta name="timeout" content="long">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/nfc_help.js"></script>
```
Comment (Honry) [inline web-nfc/NFCReader-manual.https.html]: Looks like `nfc_help.js` is never used here. Right? Same as following file.

File: web-nfc/NFCReader-manual.https.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+"use strict";
+
+promise_test(async t => {
+  const reader = new NFCReader();
+  const readerWatcher = new EventWatcher(t, reader, ["reading", "error"]);
+  reader.start();
+  const event = await readerWatcher.wait_for("reading");
+  assert_true(event instanceof NFCReadingEvent);
+}, 'Test that nfc watch success if NFC HW is enable.');
```
Comment (Honry) [inline web-nfc/NFCReader-manual.https.html]: Nit: s/enable/enabled

**Review by Honry — APPROVED** [review]
Thanks, LGTM

**Pull request merged**